### ISSUE TITLE
Arbitrary element expansion (vs. emmet-ls)

### ISSUE THREAD

**Issue by kpko** [olrtg/emmet-language-server#19]
Hi!

I've tried emmet-language-server instead of emmet-ls, because I don't like that every word I write gets expanded into a tag, even though I have a close auto complete entry that would make more sense. 

I've read here https://github.com/aca/emmet-ls/issues/55 that this language server fixes that problem, but it seems to have the same behavior: I'm editing a .tsx file, write "asdf" and it gets expanded to "<asdf></asdf>". Is there an option I didn't see?

**Comment by olrtg** [comment]
Hi @kpko! Hmmm.. that's weird, I don't get completions for that (even triggering manually with ctrl + space in my setup). Can you double-check your LSP/cmp config?

<img width="264" alt="Screenshot 2023-10-26 at 10 32 30" src="https://github.com/olrtg/emmet-language-server/assets/20072509/c766c110-b428-4d8e-bd8f-7ffff9e20897">

**Comment by olrtg** [comment]
Did you get it working? 🤔

**Comment by kpko** [comment]
Hi, so my cmp configuration looks like this:

```lua
local cmp = require'cmp'

cmp.setup({
  snippet = {
    expand = function(args)
      vim.fn["vsnip#anonymous"](args.body) 
    end,
  },
  mapping = cmp.mapping.preset.insert({
    ['<C-d>'] = cmp.mapping(cmp.mapping.scroll_docs(-4), { 'i', 'c' }),
    ['<C-f>'] = cmp.mapping(cmp.mapping.scroll_docs(4), { 'i', 'c' }),
    ['<C-Space>'] = cmp.mapping(cmp.mapping.complete(), { 'i', 'c' }),
    ['<C-y>'] = cmp.config.disable, 
    ['<C-e>'] = cmp.mapping({
      i = cmp.mapping.abort(),
      c = cmp.mapping.close(),
    }),
    ['<CR>'] = cmp.mapping.confirm({ select = true }),
    ['<TAB>'] = cmp.mapping.confirm({ select = true })
  }),
  sources = cmp.config.sources({
    { name = 'nvim_lsp' },
    { name = 'nvim_lsp_signature_help' },
    { name = 'path' },
    { name = 'vsnip' }
  }, {
    { name = 'buffer', keyword_length = 5 }
  })
})

cmp.setup.cmdline('/', {
  sources = {
    { name = 'buffer' }
  }
})
```

My lsp configuration for emmet-language-server seems very basic:

```lua
nvim_lsp.emmet_language_server.setup{
  capabilities = capabilities
}
```

But I'm using other language servers like gopls, rust_analyzer, tsserver, tailwindcss. Maybe one of them is producing this problem.

**Comment by kpko** [comment]
For a test I deactivated every language server except for emmet-language-server and I get the following output:

![image](https://github.com/olrtg/emmet-language-server/assets/835010/e9ff04d1-1704-445f-a9b9-276aaef1e1f2)

**Comment by olrtg** [comment]
When you tab (or the keybinding that you use for going to the next cmp result) how does it look like? 🤔 

<img width="364" alt="Screenshot 2023-11-10 at 10 22 05" src="https://github.com/olrtg/emmet-language-server/assets/20072509/8434a739-5d95-4dd7-8e50-d00be59b7576">

This is what I want to see ☝🏽

**Comment by kpko** [comment]
I get the html tag expansion after pressing tab:
![image](https://github.com/olrtg/emmet-language-server/assets/835010/9c79356b-379a-436b-8511-58c8a28d84d9)

For example this:
![image](https://github.com/olrtg/emmet-language-server/assets/835010/0eacd5d7-4a9d-404b-9584-98c63895f280)

Leads to:
![image](https://github.com/olrtg/emmet-language-server/assets/835010/87cdee47-0dba-4f55-9f66-a47ef6cb76bf)

**Comment by kpko** [comment]
I might have found something, this might be completely unrelated to your implementation.
![image](https://github.com/olrtg/emmet-language-server/assets/835010/db9066a2-22b0-4ce8-ba10-5d78b296d26d)

I've reinstalled your package via npm and now I get another "Comes from" in whence:
![image](https://github.com/olrtg/emmet-language-server/assets/835010/d2ec16fe-406d-4bc3-a690-7d3f65de48d5)

I needed to 'force' the install because npm would not overwrite the binary. 

Now I'm trying to find out where the old emmet-language-server came from.
